sdk/core/src/services/BloomVaultStorageService.ts

```diff
@@ -233,10 +247,35 @@ export default class BloomVaultStorageService {
   }
 
   public async searchCredentials(storageRegion: string, types?: string[][]): Promise<any[]> {
+    const doesMigrationStarted = await this._migrationHelper.doesMigrationStarted()
```

Comment:
try-catch here to avoid error and continue processing get VCs. `doesMigrationStarted` should be false by default